.github/workflows/docker.yml

```diff
@@ -60,6 +60,12 @@ jobs:
           tags: ${{ steps.meta_server.outputs.tags }}
           labels: ${{ steps.meta_server.outputs.labels }}
 
+      - name: Run Trivy vulnerability scanner on Server image
+        uses: ./.github/actions/trivy
```

Comment:
does this block PRs if it finds things? if it doesn't, and we want it to, what would change here? (or if it does, and we don't want it to, ...?)

Reply:
Currently, it fails PRs but allows us to merge anyways. We can tune this as we like.

Reply:
yeah, I was just curious what the change to block merging would look like